index.d.ts

```diff
@@ -24,8 +24,8 @@ export declare class Image {
 
     static isImage(object: any): boolean;
     static fromCanvas(canvas: HTMLCanvasElement): Image;
-    static createFrom(other: Image, options: ImageConstructorOptions);
-    static load(image: (string|ArrayBuffer|Uint8Array), options?: RequestInit)
+    static createFrom(other: Image, options: ImageConstructorOptions): Image;
+    static load(image: (string|ArrayBuffer|Uint8Array), options?: RequestInit): Image;
```

Comment:
This method returns a Promise<Image> IIRC

Reply:
Yes you're right, I overlooked that

Reply:
fixed in 10dae9be2b5ca9c71063a7b1e7db122e2dbd787a